src/main/java/duke/other/Ui.java

```diff
@@ -0,0 +1,139 @@
+package duke.other;
+
+import duke.task.TaskList;
+import duke.task.Task;
+
+import java.util.Scanner;
+
+/**
+ * Represents the UI of Duke.
+ */
+public class Ui {
+    private static final Scanner SCANNER = new Scanner(System.in);
```

Comment:
don't SCANNER! its not a constant! more like the reference to the object is constant! we tend to only use caps when the object / primitive itself is immutable! I could be wrong!